docs/self-hosted/deployment/storage/s3-bucket.md

```diff
@@ -58,3 +58,64 @@ Replace your *bucket name* in the appropriate places.
 2. Click on the user, you will be to see a screen as shown below. Copy the User ARN and paste it in the above policy.
 
 ![user-arn](./images/user-arn.png)
+
+**Update AWS S3 CORS Configuration**
+
+In order to support direct file upload from the widget and from the chatwoot dashbord we need configure CORS to the respective storage cloud.
+Refer this link for more information: https://edgeguides.rubyonrails.org/active_storage_overview.html#cross-origin-resource-sharing-cors-configuration
+
+To make CORS configuration changes on S3:
+1. Go to your S3 bucket
+2. Click on permissions tab.
+3. Scroll to Cross-origin resource sharing (CORS) and click on `Edit` and add the respective changes shown below.
+
+![aws-cors-setup](./images/aws-cors-setup.png)
+
+So for self-hosted version, you have your widget on https://www.example-site.com and your chatwoot dashboard on https://www.app.example-site.com the these both will be in AllowedOrigin array, it will look like below.
```

Comment:
@tejaswinichile Does this mean if the user has deployed the widget on for eg: google.com and the app is hosted on app.chatwoot.com, then do we need both URLs to be whitelisted here?